src/util/navigation.js

```diff
@@ -13,6 +13,7 @@ export const REQUIRED_EDIT_POST_MATCH = `:detail(post)/:postId(${POST_ID_MATCH})
 
 export const GROUP_DETAIL_MATCH = `:detail(group)/:detailGroupSlug`
 export const OPTIONAL_GROUP_MATCH = `:detail(group)?/(:detailGroupSlug)?`
+export const hyloUrlRegex = /http[s]?:\/\/hylo\.com(.*)/g // https://regex101.com/r/yQxli1/1/
```

Comment:
i think we want to use the getHost() function here so we can use/test locally and on staging. will that work with subdomains like staging.hylo.com? lets figure that out

Reply:
I'm anticipating this will work on each env, without the use of `getHost()`.

Added a note here:
https://github.com/Hylozoic/hylo-evo/pull/934/files#r695211734

Reply:
It's a quibble but wondering what not `HYLO_URL_REGEX` to match how we otherwise name constants?

Reply:
Ah yes! It only got moved there at the last moment and I missed that detail